sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/ReportImporter.java

```diff
@@ -19,12 +19,9 @@
  */
 package org.sonar.plugins.php.phpunit;
 
-import java.io.File;
 import java.util.Map;
 import org.sonar.api.batch.sensor.SensorContext;
 
-public interface PhpUnitImporter {
-
-  void importReport(File coverageReportFile, SensorContext context, Map<String, Integer> numberOfLinesOfCode);
-
+public interface ReportImporter {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Annotate the "ReportImporter" interface with the @FunctionalInterface annotation [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1609)